packages/react-next/src/components/Dropdown/Dropdown.types.ts

```diff
@@ -6,9 +6,9 @@ import { ISelectableDroppableTextProps } from 'office-ui-fabric-react/lib/utilit
 import { ResponsiveMode } from 'office-ui-fabric-react/lib/utilities/decorators/withResponsiveMode';
 import { IKeytipProps } from '../../Keytip';
 import { RectangleEdge } from 'office-ui-fabric-react/lib/utilities/positioning';
-import { ICheckboxStyleProps, ICheckboxStyles } from '../Checkbox/Checkbox.types';
-import { ILabelStyleProps, ILabelStyles } from 'office-ui-fabric-react/src/components/Label/Label.types';
-import { IPanelStyleProps, IPanelStyles } from 'office-ui-fabric-react/src/components/Panel/Panel.types';
+import { ICheckboxStyleProps, ICheckboxStyles } from 'office-ui-fabric-react/lib/Checkbox';

```

Comment:
Shouldn't this one point to the one in `@fluentui/react-checkbox`?

```suggestion
import { ICheckboxStyleProps, ICheckboxStyles } from '@fluentui/react-checkbox/lib/Checkbox';

```